tests/integration/dumpMetadata/dumpMetadata.spec.ts

```diff
@@ -259,4 +267,108 @@ describe('dumps', function () {
       });
     });
   });
+
+  describe('POST /dumps', function () {
+    describe(`${HAPPY_PATH}`, function () {
+      it('should return 201 status code and create the dump on the db', async function () {
+        const fakeDumpMetada = createFakeDumpMetadata();
+        const { id, ...dumpCreationBody } = fakeDumpMetada;
```

Comment:
update `@map-colonies/eslint-config` to `v2.2.0` to dispose eslint warning

Reply:
done.